tools/clang/test/HLSLFileCheck/hlsl/workgraph/common_input_parameters_02.hlsl

```diff
@@ -18,11 +18,11 @@
 
 // CHECK:  %[[ptr:.+]] = getelementptr %struct.RECORD.0, %struct.RECORD.0 addrspace(6)* %[[node_ptr]], i32 0, i32 0
 // CHECK:  store i32 0, i32 addrspace(6)* %[[ptr]], align 4
-// CHECK:  %[[ptr:.+]] = getelementptr inbounds %struct.RECORD.0, %struct.RECORD.0 addrspace(6)* %[[node_ptr]], i32 0, i32 1, i32 0
+// CHECK:  %[[ptr:.+]] = getelementptr {{(inbounds )?}}%struct.RECORD.0, %struct.RECORD.0 addrspace(6)* %[[node_ptr]], i32 0, i32 1, i32 0
```

Comment:
Pretty sure changes in this file are unnecessary if you only do `MergeGepUse` after `ReplaceUsesForLoweredUDTImpl` instead of before.